cg/apps/lims/api.py

```diff
@@ -53,10 +54,15 @@ def __init__(self, config):
     def user(self) -> Researcher:
         return self.get_researchers(username=self.username)[0]
 
-    def sample(self, lims_id: str):
+    def sample(self, lims_id: str) -> dict[str, Any]:
```

Comment:
```suggestion
    def get_sample(self, lims_id: str) -> dict[str, Any]:
```
Disregard if out of scope.

Reply:
We're using the sample method and the LIMS sample object interchangeably, so I'd prefer to handle it in another PR 😓